code/game/machinery/autolathe.dm

```diff
@@ -365,6 +365,7 @@
 				for(var/i=1, i<=multiplier, i++)
 					var/obj/item/new_item = new D.build_path(A)
 					new_item.materials = new_item.materials.Copy()
+					new_items.autolathed = TRUE
```

Comment:
``new_item.autolathed = TRUE``

Reply:
how do i suggest code edits help this is hard

Reply:
click the plus icon on the side

Reply:
then click the little file thingy